apps/modernization-ui/src/apps/page-builder/pages/EditPage/ManageTabs/ManageTabs.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { PagesTab, Tab } from 'apps/page-builder/generated';
+import './ManageTabs.scss';
+import { Icon } from '@trussworks/react-uswds';
+import { Icon as IconComponent } from 'components/Icon/Icon';
+
+type Props = {
+    tabs: Tab[];
+    setSelectedEditTab: (tab: PagesTab, index: number) => void;
+};
+
+const ManageTabs = ({ tabs, setSelectedEditTab }: Props) => {
+    return (
+        <div className="manage-tabs">
+            {tabs.map((tab, i) => {
+                return (
+                    <div key={i} className="manage-tabs__tile" style={{ listStyle: 'none' }}>
+                        <div className="manage-tabs__label">
+                            <IconComponent name="drag" />
+                            <IconComponent name="folder" />
+                            {tab.name}
```

Comment:
Issue: Design shows the number of elements within a tab alongside its name
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/18f497ce-c58b-4477-b112-c247d6ae1651)

Reply:
This is still being debated, like the number tiles in the Edit Page view.
Looking like users prefer the number explained but designs are still pending.